api/swagger.yml

```diff
@@ -386,6 +386,36 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Diff"
+    OtfDiff:
+      type: object
+      required:
+        - version
+        - operation
+        - timestamp
+        - operation_content
```

Comment:
TBD: do we want to add an operation-level diff type to this structure instead of mapping table operations into diff types on the client side?